desktop/containerd/index.md

```diff
@@ -209,7 +209,7 @@ View Tags on DockerHub to see multi-platform result:
 
 ## Containerd Image Store Release Notes
 
-(2022-10-19)  
+### 2022-10-19 
```

Comment:
Do we have a version number for Containerd image store?

Reply:
No, I just checked with Bruno and they're just aligning with the current DD version at the moment. The long-run plan is to move this into Docker Engine and then I imagine this will come under Docker Engine release notes.